### PR TITLE
feature: add luajit-releng tool, similarly to lua-releng.

### PR DESCRIPTION
**Pull request by doujiang24**
depends on: https://github.com/openresty/luajit2/pull/9

Only line 97 & line 98 are different from `lua-releng`.

### CODE REVIEW

File: luajit-releng [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+my (@luas, @tests);
+
+my %opts;
+getopts('Lse', \%opts) or die "Usage: lua-releng [-L] [-s] [-e] [files]\n";
+
+my $silent = $opts{s};
+my $stop_on_error = $opts{e};
+my $no_long_line_check = $opts{L};
+
+my $check_lua_ver = "luac -v | awk '{print\$2}'| grep 5.1";
```
Comment (agentzh) [inline luajit-releng]: We should replace `luac` with `luajit` here as well.

File: luajit-releng [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        }
+    }
+    if (!$found_ver) {
+        w("WARNING: No \"_VERSION\" or \"version\" field found in `$file`.");
+    }
+    close $in;
+
+    p("Checking use of Lua global variables in file $file...");
+    p("op no. line     opcode  args        ; code");
+    blank("luajit -bL $file | grep -E 'G[GS]ET' | grep -vE '\\<(require|type|tostring|error|ngx|ndk|jit|setmetatable|getmetatable|string|table|io|os|print|tonumber|math|pcall|xpcall|unpack|pairs|ipairs|assert|module|package|coroutine|[gs]etfenv|next|rawget|rawset|rawlen|select)\\>'");
```
Comment (agentzh) [inline luajit-releng]: Maybe we should also add the `arg` global to the whitelist?

File: luajit-releng [hunk 3]
```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env perl
```
Comment (agentzh) [inline luajit-releng]: Better rename the tool to `lj-releng`. I always love shorter names :)

**Comment by doujiang24** [comment]
@agentzh Fixed, thanks for your review:)

**Comment by agentzh** [comment]
This is already merged.

**Pull request closed without merging**